en/docs/guides/organization-login/try-organization-login.md

```diff
@@ -200,7 +200,7 @@ To set up the sample application:
 
 Add the following configurations to the `deployment.toml` file found in `<IS_HOME>/repository/conf/` to allow HTTP POST requests:
 
-    ``` toml
+    [cors]
     allow_generic_http_requests = true
     allow_any_origin = false
     allowed_origins = [
```

Comment:
Shall we add, "http://localhost.com:3000" also to the allowed origins?

Reply:
That is probably a good idea